models/cam-se/model_mod.f90

```diff
@@ -345,6 +349,9 @@ subroutine static_init_model()
 ! to set up what will be read into the cam state vector
 call set_cam_variable_info(cam_template_filename, state_variables)
 
+! Verify that required variables are in the state vector.
+call verify_state_var_list
```

Comment:
this is a nitpick, but I think your routine name is clear enough that you don't need this comment.

```
! this routine does stuff
call do_stuff
```